### PR TITLE
Feature nxp 20523 add guards on blob to file operation

### PR DESCRIPTION
**Pull request by egiuly-nuxeo**
_2016-09-27 4:47 PM_

JIRA : https://jira.nuxeo.com/browse/NXP-20523

_2016-09-27 5:28 PM_

New commit

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/nuxeo/nuxeo/pull/248" target="_blank">ReviewNinja

**Comment by kevinleturc** [comment]
👍

**Comment by egiuly-nuxeo** [comment]
@kevinleturc thanks for your review
I just added a new commit, to enrich my test. Can you tell me if it suits you please?
https://github.com/nuxeo/nuxeo/pull/248/commits/945e80f12bc420b60887da89bb5112243cda2ec3

File: nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/blob/BlobToFileTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            testNotAllowed(ctx, alias, params);
+        }
+        notAdminSession.close();
+    }
+
+    protected void testNotAllowed(OperationContext ctx, String id, Map<String, Object> params) throws IOException {
+        try {
+            automationService.run(ctx, id, params);
+        } catch (OperationException e) {
+            assertNotNull(e.getMessage());
```
Comment (loopingz) [inline nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/blob/BlobToFileTest.java]: If the operation succeed then you don't assert anything and will go through

Comment (egiuly-nuxeo) [inline nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/blob/BlobToFileTest.java]: Indeed, I should add a `fail(..)`

**Pull request merged**